saskatoon/harvest/admin.py

```diff
@@ -127,18 +122,26 @@ def owner_edit(self, _property):
     def harvests(self, _property):
         return _property.harvests.count()
 
+    @admin.action(description="De-authorized selected properties")
+    def reset_authorize(self, request, queryset):
+        """Set authorized=None to ACTIVE properties (False if inactive)"""
```

Comment:
Does the inactive status automatically mean that it's unauthorized ? I'm not sure, please check https://lesfruitsdefendus.readthedocs.io/fr/latest/pre-pick-guide/annexes/D-using-the-property-list.html#active-status, we can read: 

------

« Inactive »:

If a property has this value, it means that… The tree-owner hasn’t necessarily denied authorization to harvest at their property ever again. We do have a few « Inactive » properties whose owners have asked not be contacted in the future for various reasons. But the majority of our « Inactive » properties have maintained an « Inactive » status simply because the tree-owners did not explicitly authorize a harvest for 2016, the year that Saskatoon was launched. So, just to be sure, please check for notes in the « About this Property » field of the property record. Also check for notes in all harvests listed under « Harvests at this Property » (see « Reviewing Property Information » section below):

If there are any notes indicating that at some point the tree-owner asked not to be contacted in future years, do not contact them.

If, on the other hand, you don’t see any indication in the property record that a tree-owner ever specifically asked us to stop contacting them, then it’s OKAY for you to contact them and ask if they would be interested in having their fruit harvested this year! (*)

From now on, assign this value to a property if… The tree-owner has asked not to be contacted in future years. Include detailed notes indicating this and explain the situation in the « About this Property » field of the property record. We will continue to keep the property record in the database indefinitely for reference purposes, and in case the situation changes.

------

So I'm not sure this behaviour is the best, maybe let's stick to setting to `None` for if the property if active ? But I'm unsure honestly.

Reply:
> But the majority of our « Inactive » properties have maintained an « Inactive » status simply because the tree-owners did not explicitly authorize a harvest for 2016, the year that Saskatoon was launched.

Basically this is confusing and we should look into cleaning these "inactive" properties that might want to be called back if there is no comment. Let's call if you want

Reply:
> Let's call if you want

I have plans for this afternoon

Reply:
ok, I've changed it to set `authorized=None` for all selected properties